src/modules/events-app.module/module.html

```diff
@@ -1,3 +1,10 @@
+  {% require_js %}
+  <script type="application/json" data-module-instance="{{ name }}">
+      {
+        "formGuid": "{{ module.config.event_form.form_id }}"
+      }
```

Comment:
What if we did `{{ module|tojson }}`? Would it help when it comes to other aspects?

Reply:
Yeah, I had this originally and I agree it would help with other aspects-- I ran into the issue where `{{ module }}` doesn't return anything (empty dicts) for fields that don't have defaults set. 

Similar to including a form in both modules, it was a product of stringing things together to see how it all worked first. I'd like to simply pass in the whole module.

Reply:
Implemented in https://github.com/HubSpot/cms-event-registration/pull/49/commits/5c47c8024f4cc1cd918ef3ed629a02fa7cbcec05